src/styles.css

```
@@ -67,12 +67,12 @@ body.modal-open{overflow:hidden;  height:100vh;}
 @keyframes pulse{0%,100%{opacity:1;transform:scale(1)}50%{opacity:.5;transform:scale(.75)}}
 
 /* ── HERO ── */
-.hero{padding:48px 24px 36px;text-align:center;position:relative;overflow:hidden}
+.hero{padding:64px 32px 36px;text-align:center;position:relative;overflow:hidden}
 .hero::before{content:'';position:absolute;inset:0;background:radial-gradient(ellipse 70% 60% at 50% 0%,rgba(244,107,14,.07),transparent);pointer-events:none}
```

Stylelint (17.11.1)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)